### ISSUE TITLE
[BUG]: Supremium hoe dupe

### ISSUE THREAD

**Issue by TatPoi** [klikli-dev/occultism#900]
### Bug Description

I somehow duped 892k supremium hoes while making some inferium farmland. Also, if you put it into the storage like my dumb self did, it will fill up every available slot in storage. 

Pls fix, otherwise amazing mod, ty. 

### Logs and Crash Reports

N/A

### Screenshots and/or Videos

![tAEBnmkQOb](https://user-images.githubusercontent.com/27893749/224525257-1ae68b05-28f6-4298-8f72-7736001b41af.png)

![puVfK8qHpp](https://user-images.githubusercontent.com/27893749/224525216-757da760-4ccc-46d9-97c9-601f401f4a68.png)

### Bug Reproduction

1. open storage accessor
2. use the crafting recipe for mystical agriculture farmland (I checked, every essence causes this glitch)
3. there should be a stack number next to the hoe in the crafting grid after creating a stack of essence soil
4. profit!?
![08WafnMKaB](https://user-images.githubusercontent.com/27893749/224525475-a41b0895-003f-4769-931b-f239f872569d.png)

### Expected Behavior

I expected it to remain at 1 supremium hoe and not duplicate to nearly a million. 

### Minecraft Version

1.19.2

### Mod Version

1.67.0

### Modpack

ATM

**Comment by TatPoi** [comment]
Oh also if you put the stacked supremium hoes back into the recipe, it increases in size even faster. I got a stack count of 100k at one point, it did not cap at 64.

**Comment by klikli-dev** [comment]
@TatPoi thanks for reporting!
Looks like I need to do something to be compatible with the MA farmland recipe, I will check it out!

**Comment by klikli-dev** [comment]
@TatPoi hmm I just added MA in my dev env and tried your steps, but all I get is a hoe with damage haha
Will try in full ATM - I suspect there is another mod interfering somehow. If we find that it should be fixable though

**Comment by klikli-dev** [comment]
@TatPoi this does not seem to reproduce for me on ATM 8 either. I tried different MA hoes and essences, but the hoe simply loses durability. 

Does this happen for you when you create a fresh world in SP (I tried on a server, but only with occultism + MA, not yet an ATM server? It seems like - at least on my end - it does not happen on a fresh world. Maybe some additional conditions have to be met? 

EDIT: After being pointed to the discord chats discussing the issue - it seems I tested with the wrong hoes, because as you said there, the one not losing durability is the issue. I was not aware that only some MA hoes have that quality

EDIT 2: Turns out I *did* test with the supremium hoe, but in my case, if I shift-craft the hoe disappears after crafting one stack. Also not desirable, but needs further testing

EDIT 3: Thanks to the good people of the ATM team I managed to repro the dupe as well - on to fixing!
